### ISSUE TITLE
Add co-maintainer

### ISSUE THREAD

**Issue by ndmitchell** [ndmitchell/nsis#12]
I don't really use this library that much anymore. I used to use it at `$WORK` and as part of MinGHC, but I changed `$JOB` and MinGHC got discontinued. A co-maintainer who actually used the library would be very beneficial.

CC @domenkozar and @jmitchell as two recent pull request authors who seem to have opinions about how it should be done.

**Comment by domenkozar** [comment]
I'm willing to be one, given that we're now using the library.

**Comment by jmitchell** [comment]
I'm happy to help too. :)

**Comment by ndmitchell** [comment]
Thanks for the offers. I've given you both permissions to write to this repo. I'm happy to let the library sit as it is, but if you think it needs additions/extensions/big changes just raise tickets or patches and we can figure it out together. I'll announce it on my blog when I get around to it.

**Comment by jmitchell** [comment]
@ndmitchell, do you tend to merge all new work into master and bump the version and tag only once you're ready to make a release?

**Comment by ndmitchell** [comment]
@jmitchell Yep, pretty much. I have a list of my general development style/guidelines/preferences at https://github.com/ndmitchell/neil/blob/master/README.md - I just added a note about merging into `master`.
